protocol/protobuf/chat_message.proto

```diff
@@ -54,6 +54,19 @@ message DeleteMessage {
   MessageType message_type = 5;
 }
 
+message DeleteForMeMessage {
+  uint64 clock = 1;
+
+  string chat_id = 2;
+  string message_id = 3;
+
+  // Grant for community delete messages
+  bytes grant = 4;
```

Comment:
Fixed with 
```go
message DeleteForMeMessage {
  uint64 clock = 1;
  string message_id = 2;
  bool undo = 3;
}
```
`undo` is for the undo feature

Reply:
I think `undo` it should be only a local action, basically we should not dispatch the message until the `undo` has timed out, for example:
1) User deletes message
2) The client will update the UI, and show the undo button
3) If the undo button is not clicked, it will then make a query to status-go and actually delete the message

The issue is that if the app crashes/ is closed, the action won't be performed, but I think that's preferrable to send `undo` over the wire (unless you can always undo, but as I understand it's only for a limited period, i.e 3/5 seconds),
what do you think?

Reply:
Considering the delete for everyone feature, undo action should definitely guard by time. And we better keep these two undo consistent. 
I'll change the code.